src/kmer.rs

```diff
@@ -154,10 +155,10 @@ mod tests {
         let k_iter = Kmers::new(b"AGNCT", 2);
         for (i, k) in k_iter.enumerate() {
             match i {
-                0 => assert_eq!(k, &b"AC"[..]),
-                1 => assert_eq!(k, &b"CN"[..]),
-                2 => assert_eq!(k, &b"NG"[..]),
-                3 => assert_eq!(k, &b"GT"[..]),
+                0 => assert_eq!(k, b"AG"),
+                1 => assert_eq!(k, b"GN"),
+                2 => assert_eq!(k, b"NC"),
+                3 => assert_eq!(k, b"CT"),
                 _ => unreachable!("Too many kmers"),
```

Comment:
I'm especially pointing out these changes. Unlike the others, the values themselves are actually being changed, not just the extra slicing/referencing syntax.